docs/schema/action-element-actions-elementworkflowinfo-elementaction4.md

```diff
@@ -199,8 +200,6 @@ If the schema defines specific requirements, such as **sequence**, **minOccurs**
 <col width="15%" />
 <col width="35%" />
 <col width="20%" />
```

Comment:
Should we also get rid of lines 197 through 202?

Reply:
no it's not part of the request or the validation. I got overzealous and was jumping the gun. I'll put the closing tag back.